### PR TITLE
210822 Quality Control (for Belarus2019 and Chad2019) JW

### PR DESCRIPTION
**Pull request by jianingwwww**
- rename child anthro variables to match with the variable name in the quality control do file

Based on the raw data of Belarus2019 and Chad2019, I adjusted the codes to make sure that as many indicators as possible can be generated smoothly. 
Besides, could you please tell me which folder in the OneDrive should I upload the indicator file to?

### CODE REVIEW

File: 06_Prepare_MICS6_01_Batch_final.do [hunk 1]
```diff
@@ -595,14 +595,20 @@ clear
         label var gl_adm1_code "Global Administrative Unit Layers(GAUL) Code"
 		label var gl_adm0_code "Global Administrative Unit Layers(GAUL) Country Code"
 		
-	rename (c_del_eff1 c_del_eff1_q c_del_eff2 c_del_eff2_q w_unmet c_measles_vacc c_mateduc ind_sampleweight mor_wdob mor_doi)(c_sba_eff1 c_sba_eff1_q c_sba_eff2 c_sba_eff2_q w_unmet_fp c_measles c_mateduc w_sampleweight hm_dob hm_doi)
+	rename (c_del_eff1 c_del_eff1_q c_del_eff2 c_del_eff2_q w_unmet c_measles_vacc c_mateduc ind_sampleweight mor_wdob mor_doi)(c_sba_eff1 c_sba_eff1_q c_sba_eff2 c_sba_eff2_q w_unmet_fp c_measles w_mateduc w_sampleweight hm_dob hm_doi)
```
Comment (wengxyu1030) [inline 06_Prepare_MICS6_01_Batch_final.do:598]: Please keep c_matedu as it is, and change the variable name w_matedu to c_matedu in the quality control file instead.

**Review by wengxyu1030 — CHANGES_REQUESTED** [review]

**Comment by wengxyu1030** [comment]
> * rename child anthro variables to match with the variable name in the quality control do file
> 
> Based on the raw data of Belarus2019 and Chad2019, I adjusted the codes to make sure that as many indicators as possible can be generated smoothly.
> Besides, could you please tell me which folder in the OneDrive should I upload the indicator file to?

Hi @jianingwwww , 

Thanks for your update! I have made small code change comment in the file. 

Moreover, please create a folder named "inter" under the current folder where you saved the output microdata. Kindly save the quality control result files, which you referred to as indicator files, to this new folder. 

Regards, 
Aline

**Comment by jianingwwww** [comment]
> > * rename child anthro variables to match with the variable name in the quality control do file
> > 
> > Based on the raw data of Belarus2019 and Chad2019, I adjusted the codes to make sure that as many indicators as possible can be generated smoothly.
> > Besides, could you please tell me which folder in the OneDrive should I upload the indicator file to?
> 
> Hi @jianingwwww ,
> 
> Thanks for your update! I have made small code change comment in the file.
> 
> Moreover, please create a folder named "inter" under the current folder where you saved the output microdata. Kindly save the quality control result files, which you referred to as indicator files, to this new folder.
> 
> Regards,
> Aline

Hi @wengxyu1030 , 

I have changed the variable name back and created the folder just like this:
165c8156037d4092281dee43bc70f52
Please tell me if there are further questions!

Best,
Jianing

**Comment by wengxyu1030** [comment]
Thanks @jianingwwww , I have no further question about this pull request. Please go ahead with the quality checking process with Belarus2019 and Chad2019. 

Regards, 
Aline

**Pull request merged**